docs/examples/csfle-explicit_encryption.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+use MongoDB\BSON\Binary;
+use MongoDB\Client;
+use MongoDB\Driver\ClientEncryption;
+
+require __DIR__ . '/../../vendor/autoload.php';
+
+$uri = getenv('MONGODB_URI') ?: 'mongodb://127.0.0.1/';
+
+// Generate a secure local key to use for this script
+$localKey = new Binary(random_bytes(96));
+
+/* Create a client with no encryption options. Additionally, create a
+ * ClientEncryption object to manage data keys. */
+$client = new Client($uri);
+
+$clientEncryption = $client->createClientEncryption([
+    'keyVaultNamespace' => 'encryption.__keyVault',
+    'kmsProviders' => [
+        'local' => ['key' => $localKey],
+    ],
+]);
+
+/* Drop the key vault collection and create an encryption key. Alternatively,
+ * this key ID could be read from a configuration file. */
+$client->selectCollection('encryption', '__keyVault')->drop();
+$keyId = $clientEncryption->createDataKey('local');
+
+// Select and drop a collection to use for this example
+$collection = $client->selectCollection('test', 'coll');
+$collection->drop();
+
+// Insert a document with a manually encrypted field
+$encryptedValue = $clientEncryption->encrypt('mySecret', [
+    'algorithm' => ClientEncryption::AEAD_AES_256_CBC_HMAC_SHA_512_DETERMINISTIC,
+    'keyId' => $keyId,
+]);
+
+$collection->insertOne(['encryptedField' => $encryptedValue]);
```

Comment:
Reading this example, I had to scroll up to find which client was used. You may add a comment that we can insert the value with the unencrypted client.

Reply:
I'll rename `$collection` in this example to `$unencryptedCollection`, as is done in `queryable_encryption-automatic.php`.

`csfle-explicit_encryption.php` and `csfle-explicit_encryption_automatic_decryption.php` only create a single client, so there is no ambiguity there.

Reply:
> `csfle-explicit_encryption.php` and `csfle-explicit_encryption_automatic_decryption.php` only create a single client, so there is no ambiguity there.

In facts, there is 2 variables: `$client` and `$clientEncryption`. Both are used.

Reply:
Those scripts have one Client and ClientEncryption objects, which differs from other scripts that create multiple clients. In the scripts with multiple Client objects, I've revised the examples to prefix the collection variable with "unencrypted" or "encrypted" accordingly.